### ISSUE TITLE
Checked Checkbox is too big

### ISSUE THREAD

**Issue by deeconwastaken** [nextcloud/notes#410]
**Describe the bug**
The icon for a checked checkbox looks not good

**To Reproduce**
* make a list with checkboxes
```
[ ] Butter
[ ] 2 Kästen Mineralwasser
```

* click a checkbox
```
[x] Butter
[ ] 2 Kästen Mineralwasser
```

**Expected behavior**
The checkbox should look good

**Screenshots**
![image](https://user-images.githubusercontent.com/13451395/69146906-1e3d5500-0ad1-11ea-8b2c-278a0e449f64.png)

**Server (please complete the following information):**
- Notes app version: 3.0.3
- Nextcloud version: 17.0.1
- OS: Debian
- Web server:  Apache
- PHP version: 7.2
- Database: mySQL

**Client (please complete the following information):**
 - Browser (incl. version): Chrome 78.0.3904.97 and Microsoft Edge 44.18362.449.0
 - OS: Windows 10 v1909

**Comment by korelstar** [comment]
Was this issue introduced with updating to Windows 10 1909? Do you know if it also occurred on 1903?

My virtual machine is still on 1903 (no update available) and there, it looks fine (with both Chrome 78.0.3904.108 and Edge): 
![Screenshot](https://user-images.githubusercontent.com/6277619/69160687-48e7d780-0aea-11ea-8f0d-5b9fe39dede9.png)

**Comment by korelstar** [comment]
I finally got Windows 10 1909 on a computer. But here, with Microsoft Edge 44.18362.449.0, it looks exactly as intended (see my screenshot above).

Maybe the errornous design at your computer is caused by a third-party software? Do you have the ability to test with another computer?

**Comment by Larsene** [comment]
same problem here (same apparence on Opera / Win 10 2004)
![flask](https://user-images.githubusercontent.com/8544729/70864573-86a61780-1f53-11ea-96b1-3546fb82b7cc.jpg)

**Comment by stefan-niedermann** [comment]
Can @Larsene or  @deeconwastaken Press "F12" and inspect one of the checkbox elements? A screenshot of the right side of the developer panel, especially the "computed" section would be very interesting...

**Comment by deeconwastaken** [comment]
Hello @stefan-niedermann,

just write if you need more infos.

![Anmerkung 2019-12-16 083927](https://user-images.githubusercontent.com/13451395/70888109-d3cfcb00-1fdf-11ea-8c60-a803aa9314ba.jpg)
![Anmerkung 2019-12-16 0839887](https://user-images.githubusercontent.com/13451395/70888120-d7635200-1fdf-11ea-96e6-515b834558ea.jpg)

**Comment by stefan-niedermann** [comment]
It seems, that on the `::before`-element, there is no explicit set `width` nor `height`. Therefore, it inherits those parameters from the parent, which would explain the calculated `width` to be `479.6px`.

Maybe explicitely setting those properties might help?

Unfortunately i am only running linux machines, therefore i can not test and verify this.

**Comment by stefan-niedermann** [comment]
@deeconwastaken if you know how to use the developer tools: could you set in the "Styles" tab next to "Computed" at the `::before`-Element explicitely the `width` and `height` for testing purposes? It should roughly look like this:

![grafik](https://user-images.githubusercontent.com/4741199/70890045-58bce380-1fe4-11ea-9e0c-9f078807a28f.png)

**Comment by Larsene** [comment]
@stefan-niedermann : it won't change anything. The size of the checked box is defined by the font-size property, and with the same font size, the boxes does'nt have the same rendered sizes...

![flask2](https://user-images.githubusercontent.com/8544729/70894374-2bc0fe80-1fed-11ea-807b-b3ba843bc4e4.jpg)

But if you tweaks it here, it (almost) works. But need to be tested on other operating systems :
![Uploading flask3.jpg…]()

**Comment by korelstar** [comment]
@Larsene Thanks for your investigation. However, adjusting the font-size to 1.1em breaks the checkbox size on other systems:

![Screenshot](https://user-images.githubusercontent.com/6277619/71289172-bcd90200-236c-11ea-823a-a01fe67ecc84.png)

I did some research on this. It looks that Nextcloud 17 switched from an explicit "Nextcloud font" (which was Nunito before) to platform dependent native fonts. This means, that Nextcloud now looks completely different on different operating systems. I'm wondering about this, but it's intended: https://github.com/nextcloud/server/pull/16055.

| . | Nextcloud 15 | Nextcloud 18 beta 3 | 
|---|---|---|
| Windows 10 1909 | ![Screenshot](https://user-images.githubusercontent.com/6277619/71292140-0d079280-2374-11ea-923d-f701a5dc7790.png) | ![Screenshot](https://user-images.githubusercontent.com/6277619/71292239-1bee4500-2374-11ea-9e8e-6a42a3a6adfd.png) |

The problem is, that the font used under Windows 10 1909 has a different size for unchecked vs. checked checkboxes (Unicode character 2610 vs. 2611, see screenshot above). Currently, I don't have any idea how to fix this. I would say it's a bug in that Windows font.

Do you have any ideas on this @jancborchardt ?

**Comment by Larsene** [comment]
@korelstar : you're totally right. This is not the solution (as it goes again the Nextcloud policy to be based on system font), but for users affected, you can explicitely change the font used to display the checkbox in the CustomCSS app : 
```
.CodeMirror .cm-formatting-task.cm-meta::before, .CodeMirror .cm-formatting-task.cm-property::before {
    font-family: arial;
}
```

**Comment by korelstar** [comment]
Thanks, I think it's a good approach to fix the checkbox design. I found out that the tasks app recently switched to a common Nextcloud checkbox icon: https://github.com/nextcloud/tasks/issues/506
Let's adopt this! :-)

**Comment by korelstar** [comment]
Fix is in #419. Please test this, I will have to do tests, too.
